opengvl/clients/__init__.py

```diff
@@ -5,11 +5,15 @@
 from opengvl.clients.gemma import GemmaClient
 from opengvl.clients.kimi import KimiThinkingClient
 from opengvl.clients.openai import OpenAIClient
+# from opengvl.clients.glm import GLMClient
```

Comment:
this is under dev still, ok

Reply:
I will leave it as it is for now, and I will update env in the next PR